recipes/newrelic/apm/java/linux.yml

```diff
@@ -89,15 +89,17 @@ install:
           YELLOW='\033[0;33m'
           NOCOLOR='\033[0m'
           echo -e "${YELLOW}
-          ================================================================================================
-          =                                                                                              =
-          =                                           Warning                                            =
-          =                                                                                              =
-          =         This installation will periodically check for new Tomcat processes to monitor.       =
-          =    Any new Tomcat processes started after this installation will report data to New Relic.   =
-          =                                                                                              =
-          =                                                                                              =
-          ================================================================================================
+          ==================================================================================================
```

Comment:
@mbruzina is this layout a bit scary for users? Should we have a more consistent layout with other recipe execution?

Reply:
will do in another ticket